src/main/java/com/aol/cyclops/control/ReactiveSeq.java

```diff
@@ -2840,17 +2840,21 @@ default T single(Predicate<? super T> predicate) {
 	 * 
 	 */
 	default <R> ReactiveSeq<R> retry(Function<? super T, ? extends R> fn) {
+		return retry(fn, 7, 2, TimeUnit.SECONDS);
+	}
+	
+	default <R> ReactiveSeq<R> retry(Function<? super T, ? extends R> fn, int retries, long delay, TimeUnit timeUnit) {
```

Comment:
Thanks Nikita, this is great. Could we also get some javadoc that includes an example usage code snippet?